src/main/java/Board.java

```diff
@@ -1,4 +1,105 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class Board {
-    public static String[] grid = new String[9];
+
+    private List<String> cells = Arrays.asList("", "", "", "", "", "", "", "", "");
+
+    public List<String> getCells() {
+        return cells;
+    }
+
+    public boolean gridIsEmpty() {
+       List<String> currentBoard = getCells();
+       for (int i = 0; i < currentBoard.size(); i++) {
+           if (currentBoard.get(i) != "") {
+               return false;
+           } else {
+               continue;
```

Comment:
You could actually remove the 'else' clause here - as you will return early if the board has a blank mark, otherwise the loop will continue anyway.